packages/bot-skeleton/src/constants/messages.ts

```diff
@@ -26,6 +26,8 @@ export const message_types = Object.freeze({
     SUCCESS: 'success',
 });
 
+export type TErrorTypes = 'recoverable_errors' | 'unrecoverable_errors';
```

Comment:
@sandeep-deriv Bhai can we go with enum in here as we need these strings to be static?

Reply:
this is done, thanks @shafin-deriv